openedx/core/djangoapps/user_api/accounts/tests/test_views.py

```diff
@@ -20,6 +21,7 @@
 from common.djangoapps.student.models import PendingEmailChange, UserProfile
 from common.djangoapps.student.models_api import do_name_change_request, get_pending_name_change
 from common.djangoapps.student.tests.factories import TEST_PASSWORD, RegistrationFactory, UserFactory
+from openedx.core.djangoapps.user_api import accounts
```

Comment:
where is this import being used?

Reply:
L493 `accounts.RETIRED_EMAIL_MSG`

Reply:
Constants here are all used in this way. written this to keep the consistency.

Reply:
While I am up for consistency, this is not a good pattern, as mentioned by @ansabgillani.

Reply:
changed